content/wiki/autotest-integration.md

```diff
@@ -53,9 +53,11 @@ autotest: --color
 autotest-all: --color
 ```
 
-Please see [Running Features](/configuration/#running-features) for more information about profiles, and the [Ruby on Rails](/implementations/ruby/ruby-on-rails/) page for extra help when running in that context.
+Please see [Running Features](/running/) for more information about profiles, and the 
```

Comment:
Change link "[Running Features]" to "Running Cucumber"?

Reply:
Done